homeassistant/components/iron_os/select.py

```diff
@@ -168,9 +168,7 @@ def __init__(
         entity_description: IronOSSelectEntityDescription,
     ) -> None:
         """Initialize the select entity."""
-        super().__init__(
-            coordinator.live_data, entity_description, entity_description.characteristic
-        )
+        super().__init__(coordinator.live_data, entity_description)
 
         self.settings = coordinator.settings
```

Comment:
Would it not make sense to rename this argument?
`coordinator` doesn't seem to be the right name anymore.
Maybe `coordinators` is better

```suggestion
        self.settings = coordinators.settings
```